internal/generator/vector/filter/openshift/viaq/normalize.go

```diff
@@ -80,7 +80,7 @@ if starts_with(pod_name, "eventrouter-") {
   }
 }
 `
-	RemoveStream       = `del(.stream)`
+	RemoveStream       = `.kubeernetes.container_iostream = del(.stream)`
```

Comment:
Typo here:
`kubernetes`
```suggestion
	RemoveStream       = `.kubernetes.container_iostream = del(.stream)`
```

Reply:
This is replaced during rebase